Task12-whack-a-mole/index.html

```diff
@@ -0,0 +1,72 @@
+<!-- 
+    All Rights Reserved
+    This software is property information of AKUREY S.A.
+    Contact us at contact@akurey.com
+    @summary short description for the file
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>HTML + JS + CSS</title>
+  <link rel="stylesheet" href="src/style/style.css">
+</head>
+
+<body>
+  <h1>Whack-a-mole! <span class="score">0</span></h1>
+  <div class="gameInfo">
+    <div class="scorecard">
+      <h2>Top 5 Best Scores:</h2>
+      <div class="scores"></div>
+    </div>
+    <div class="levelcard">
+      <h2>Level: <span class="level">1</span></h2>
+    </div>
+    <div class="timecard">
+      <h2>Time: <span class="time">10</span>segs</h2>
+    </div>
+  </div>
+  <div class="game">
+    <div class="hole hole1">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole2">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole3">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole4">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole5">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole6">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole7">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole8">
+      <div class="mole"></div>
+    </div>
+    <div class="hole hole9">
+      <div class="mole"></div>
+    </div>
```

Comment:
I would recommend to move this repeated code to JS if you want. Have you heard of template literals?
You can select the parent element game and inject a generated html code to it, you can use template literals, so you can create a loop to create the 9 holes into a string:
[...Array(9)].forEach((_, index) => {
  htmlToBeInserted += `<div class="hole hole${index}">
      <div class="mole"></div>
    </div>`
});

parentElement.insertAdjacentHTML('afterend',  htmlToBeInserted);